src/api_manager/service_control/aggregated.cc

```diff
@@ -93,6 +93,16 @@ const char servicecontrol_service[] =
 const char quotacontrol_service[] =
     "/google.api.servicecontrol.v1.QuotaController";
 
+// Define network failure error codes:
+// All 500 Http status codes are marked as network failure.
```

Comment:
500 causes ambiguity of `Internal Server Error`. Please use 5XX

Reply:
Done